Makefile

```diff
@@ -210,7 +210,7 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 	operator-sdk bundle validate ./bundle
 
 .PHONY: bundle-build
-bundle-build: ## Build the bundle image.
+bundle-build: bundle ## Build the bundle image.
```

Comment:
Is this a change we're replicating across operators?  Operator-SDK doesn't scaffold it this way.  It seems we have a few operators with this extra change, but most currently do not have it.

Reply:
Yeah I'm adding it across operator repos, as you can't run 'bundle-build' without running 'bundle' target (i.e there is no default bundle.Dockerfile).  Also, it's better to generate it rather than use a default. We can probably remove the default bundle.Dockerfile in repos where they exist.

Reply:
Ack

Reply:
@dprince I thought I saw it in some repos. But can't find one other than the custom one used in openstack-operator. I think the issue I faced is it always used the one locally when running bundle-build rather than regenerating one.

Reply:
@rabi so one use case to think about here is I often generate the bundle, mess with it, and then run a bundle build. I could do that manually (essentially manually build the Dockerfile). There isn't a whole lot of value in the bundle-build task as is...

Reply:
The simple workflow for me after any PR to deploy with OLM involves, docker-build/docker-push, bundle-build/bundle-push, catalog-build/catalog-push. Do you think that's incorrect? Should not that Dockerfile be generated automatically every time without needing manual intervention(?)

Reply:
@dprince I can remove that change from the PR if you think we should not change it. I can make it work locally for me with some hack.

Reply:
OK, removed it. I know it won't apply to everyone, but I guess there is a default workflow where folks don't tweak generated files.